dll/win32/netid/lang/ro-RO.rc

```diff
@@ -1,7 +1,8 @@
 /*
- * FILE:       dll/win32/netid/lang/ro-RO.rc
- *             ReactOS Project (https://reactos.org)
- * TRANSLATOR: Ștefan Fulea (stefan dot fulea at mail dot com)
+ * FILE:          dll/win32/netid/lang/ro-RO.rc
+ *                ReactOS Project (https://reactos.org)
+ * TRANSLATORS:   Ștefan Fulea (stefan dot fulea at mail dot com)
+ *                Andrei Miloiu (miloiuandrei@gmail.com)
```

Comment:
You may also add the years of contribution here (say, the way you did it [here](https://github.com/reactos/reactos/blob/master/dll/cpl/input/lang/ro-RO.rc#L6)).

Reply:
I'm not sure I added the right years for you.